dotnet/src/Connectors/Connectors.AzureOpenAI/Connectors.AzureOpenAI.csproj

```diff
@@ -0,0 +1,34 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <!-- THIS PROPERTY GROUP MUST COME FIRST -->
+    <AssemblyName>Microsoft.SemanticKernel.Connectors.AzureOpenAI</AssemblyName>
+    <RootNamespace>$(AssemblyName)</RootNamespace>
+    <TargetFrameworks>net8.0;netstandard2.0</TargetFrameworks>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <NoWarn>$(NoWarn);NU5104;SKEXP0001,SKEXP0010</NoWarn>
+    <EnablePackageValidation>false</EnablePackageValidation>
+  </PropertyGroup>
```

Comment:
I don't remember the decision about this if any was made, but do we need version suffix for this package like `<VersionSuffix>alpha</VersionSuffix>` or `<VersionSuffix>preview</VersionSuffix>`?

Reply:
It seems we've discussed it and decided to go straight to the released one. cc: @markwallace-microsoft

Reply:
This package will replace code that is already at 1.x level so we need to be back at that level when we merge this back to main. If we decide to release from this branch we may add a suffix but we can add that if needed later.